### PR TITLE
add time entry docs

### PR DESCRIPTION
**Pull request by tschuy**

### CODE REVIEW

**Comment by tschuy** [comment]
@Kennric @LyonesGamer

File: source/usage/times.rst [hunk 1]
```diff
@@ -3,3 +3,25 @@
 ============
 Time Entries
 ============
+
+TimeSync entries have a few main fields:
+
+* ``duration``: the amount of time worked
```
Comment (MorganEPatch) [inline source/usage/times.rst]: What is this in? Seconds, minutes, hours?

**Comment by tschuy** [comment]
@Kennric @LyonesGamer

**Comment by MorganEPatch** [comment]
If we're all agreed that `duration` should be in seconds, then I have no other complaints. +1

**Comment by iankronquist** [comment]
This appears to be the de facto standard already. +1 from me an merging.

**Pull request merged**